RichEditorView/Classes/RichEditorView.swift

```diff
@@ -439,6 +445,75 @@ extension RichEditorView: UIGestureRecognizerDelegate {
 // MARK: - Utilities
 extension RichEditorView {
     
+    /** Sets content offset for scrollView based on caret location **/
+    
+    private func fixScrollView(editor: RichEditorView) {
```

Comment:
This is an instance method, you don't need to pass in the `RichEditorView`. 

Also `fixScrollView()` is pretty vague. Maybe something like `scrollCursorIntoView()` ? I don't know, maybe there's something better, but it'd be nice to have something more clear about what it does.